libibverbs/verbs.h

```diff
@@ -681,6 +681,25 @@ struct ibv_mr {
 	uint32_t		rkey;
 };
 
+enum  ibv_reg_mr_in_mask {
+	IBV_REG_MR_MASK_IOVA = 1 << 0,
+	IBV_REG_MR_MASK_ADDR = 1 << 1,
+	IBV_REG_MR_MASK_FD = 1 << 2,
+	IBV_REG_MR_MASK_FD_OFFSET = 1 << 3,
+	IBV_REG_MR_MASK_DMAH = 1 << 4,
+};
+
+struct ibv_reg_mr_in {
```

Comment:
Are we still able to change the name of this structure?  Has there been a new release of libibverbs?

Adding UET support to verbs requires extending this structure, and I would also like to use it to return output data, to avoid needing a reg_mr_ex2 call or other changes.

Reply:
It has a comp_mask field for future extension.

What is the motivation to rename it ? any other suggested name ?

Note:
Even if upon success it will hold some output fields in the future, (what is the use case ?) it still can be considered as the input struct for the API which holds both the in/out data.

Reply:
Structure that provides in and out fields can't be called struct XXX_in {}.

Thanks

Reply:
It wasn't intended for output.

Let's see the use case and then re-consider whether we really need a rename.